jenkins/pull_request_creator.py

```diff
@@ -121,3 +120,62 @@ def create(self, delete_old_pull_requests):
             self.delete_old_pull_requests()
 
         self._create_new_pull_request()
+
+
+@click.command()
+@click.option(
+    '--repo-root',
+    type=click.Path(exists=True, dir_okay=True, file_okay=False),
+    required=True,
+    help="Directory containing local copy of repository"
+)
+@click.option(
+    '--base-branch-name',
+    required=True,
+    help="Base name for branch to create. Full branch name will be like jenkins/BASENAME-1234567."
+)
+@click.option('--commit-message', required=True)
+@click.option('--pr-title', required=True)
+@click.option('--pr-body', required=True)
+@click.option(
+    '--user-reviewers',
+    multiple=True,
+    required=True
+)
+@click.option(
+    '--team-reviewers',
+    multiple=True,
+    required=True
+)
+@click.option(
+    '--delete-old-pull-requests/--no-delete-old-pull-requests',
+    default=True,
+    help="If set, delete old branches with the same base branch name and close their PRs"
+)
+def main(
+    repo_root, base_branch_name,
+    commit_message, pr_title, pr_body,
+    user_reviewers, team_reviewers,
+    delete_old_pull_requests
+):
+    """
+    Create a pull request with these changes in the repo.
+
+    Required environment variables:
+
+    - GITHUB_TOKEN
+    - GITHUB_USER_EMAIL
+    """
+    creator = PullRequestCreator(
+        repo_root=repo_root,
+        branch_name=base_branch_name, commit_message=commit_message,
+        pr_title=pr_title, pr_body=pr_body,
+        # TODO: Accept lists in constructor, too (requires changes to existing callers)
```

Comment:
I need to better understand this.

Reply:
There's a data type flip-flop. The new CLI accepts a list, then packs it together as a comma-delimited string to pass to the constructor. The PR creator then splits on the commas (in `_create_new_pull_request`) so that it has a list again!

Fixing it would involve fixing the callers, some of which have config in a different repository.

Reply:
Can the constructor be updated to accept either a comma-delimited string or a list?  Then you wouldn't have to pack/unpack and you wouldn't need this TODO message.  Thoughts?

Reply:
I think I'm just going to change it to be a comma-delimited string the whole way through, for simplicity.